app/src/components/common/TmPage.vue

```diff
@@ -1,101 +1,43 @@
 <template>
-  <div class="tm-page">
-    <TmPageHeader v-if="!hideHeader" :tabs="tabs">
-      <h2 v-if="title" slot="title">
-        {{ title }}
-      </h2>
-      <h3 v-if="subtitle" slot="subtitle">
-        {{ subtitle }}
-      </h3>
-      <slot slot="menu-body" name="menu-body">
-        <TmBalance v-if="session.signedIn" />
-      </slot>
-      <slot slot="header-buttons" name="header-buttons" />
-    </TmPageHeader>
-    <main
-      class="tm-page-main"
-      :class="{ 'dark-background': darkBackground && session.signedIn }"
-    >
-      <CardSignInRequired v-if="signInRequired && !session.signedIn" />
-      <template v-else-if="managed">
```

Comment:
there may be some issues related to removing this - would be helpful if you manually QA'd this by clicking around and seeing if there are any strange behaviours.

Reply:
I saw that margins are suddenly disappearing in really thin screens like you can see here:

![image](https://user-images.githubusercontent.com/40721795/87166972-9bba0d80-c2cc-11ea-9d59-55fb4362ef50.png)

But I don't know if this is TmPage refactor related. Networks and the rest are doing fine. It is only Proposals it seems

Reply:
which margins?

Reply:
on the sides

Reply:
i think that's how it is in production already - not caused by this change